packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -124,7 +124,7 @@ export class MultiSelectExample extends React.PureComponent<ExampleProps, MultiS
 
         return (
             <Example options={this.renderOptions()} {...this.props}>
-                <MultiSelect<Film>
```

Comment:
The omission of the specified `Film` generic parameter to the component is necessary here, otherwise we'll get a type error:
```
'readonly Film[]' is 'readonly' and cannot be assigned to the mutable type 'Film[]'
```
Another way to solve this is to explicitly define the second type parameter, e.g.
```ts
<MultiSelect<Film, readonly Film[]>
```